### ISSUE TITLE
v3.19.0 test failure building on i686

### ISSUE THREAD

**Issue by mefuller** [go-task/task#949]
Building v3.19.0 for Fedora 37 on i686 architecture fails with the following error:

```
Testing    in: /builddir/build/BUILD/task-3.19.0/_build/src
         PATH: /builddir/build/BUILD/task-3.19.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/task-3.19.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags " -X github.com/go-task/task/version=3.19.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -Wl,--build-id=sha1 -specs=/usr/lib/rpm/redhat/redhat-package-notes '"
      testing: github.com/go-task/task
github.com/go-task/task
--- FAIL: TestFileWatcherInterval (0.72s)
    task_test.go:1699: 
        	Error Trace:	/builddir/build/BUILD/task-3.19.0/_build/src/github.com/go-task/task/task_test.go:1699
        	Error:      	Not equal: 
        	            	expected: "task: Started watching for tasks: default\ntask: [default] echo \"Hello, World!\"\nHello, World!\ntask: [default] echo \"Hello, World!\"\nHello, World!"
        	            	actual  : "task: Started watching for tasks: default\ntask: [default] echo \"Hello, World!\"\nHello, World!"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -3,3 +3 @@
        	            	 Hello, World!
        	            	-task: [default] echo "Hello, World!"
        	            	-Hello, World!
        	Test:       	TestFileWatcherInterval
FAIL
exit status 1
FAIL	github.com/go-task/task	1.739s
```

- Task version: 3.19.0
- Operating System: Fedora 37
- Complete build log: https://koji.fedoraproject.org/koji/getfile?taskID=95025409&volume=DEFAULT&name=build.log

The above error is not seen when building v3.19.0 on F38/rawhide

**Comment by andreynering** [comment]
Hi @mefuller,

That's an intermittent test. I noticed it has been failing once in a while on our CI as well. I certainly need to fix or temporarily disable it.

In the meantime, a retry should work for you.

**Comment by mefuller** [comment]
Thanks for the update - closing

**Comment by andreynering** [comment]
Addressed at d6d51a2f8becb72f64cbba7e778c29ba85cefb38
